### PR TITLE
Add a link back to the app via a breadcrumb.

### PR DESCRIPTION
**Pull request by delynn**
An alternate approach that uses the existing breadcrumbs and also has a really sensible default.

screen shot 2015-12-04 at 5 04 13 pm

### CODE REVIEW

**Comment by delynn** [comment]
@jnunemaker Sorry this took so long, but life got a little hectic for me.

I think I prefer the way this turned out vs. #89.

File: lib/flipper/ui/action.rb [hunk 1]
```diff
@@ -159,6 +162,10 @@ def active?
       # href - The String href for the anchor tag (optional). If nil, breadcrumb
       #        is assumed to be the end of the trail.
       def breadcrumb(text, href = nil)
+        if href != nil
```
Comment (jnunemaker) [inline lib/flipper/ui/action.rb]: I would do the more idiomatic `unless href.nil?`.

**Comment by jnunemaker** [comment]
Tweaked a few things: https://github.com/jnunemaker/flipper/compare/48b073006604a0671bbfcfa95a8d9e632b660c41...0c5b88232e60d9da5598c6fb11a806af89087f29

**Comment by delynn** [comment]
Looks great. Thanks!

**Pull request merged**